### ISSUE TITLE
Scroll and Wand items should be generated with the tradition traits of the spell they contain

### ISSUE THREAD

**Issue by mechamaya** [foundryvtt/pf2e#13625]
Currently when a scroll is generated, it takes on the traits of the spell it contains. But since spell traditions are no longer in the traits section of the item, they don't get applied to the scroll. This means Identify Magic doesn't see it as an item of a specific tradition. Same thing happens with wands.

**Comment by bcronin90** [comment]
This is technically correct, if inconvenient. A scroll does not have a tradition attached to it and a spell cast from it only gains the tradtition trait upon casting. (GM Core 262)

**Comment by stwlam** [comment]
Wouldn't that make this proposal technically incorrect but convenient?

**Comment by mechamaya** [comment]
Yeah after we discussed this on Discord, this is true. Scrolls and Wands do not RAW have the traits of their traditions. And it's not clear if this is supposed to be used for Identify Magic (though I would think it does). It just would not use traits, just the traditions of the spell. However for foundry to know this, it would either have to throw traits on them (which goes against RAW), or overhaul identify magic to be aware of the spell contents of the scroll/wand (which sounds more difficult).

So yeah I would agree, it's incorrect but convenient. Shark is right.

**Comment by nikolaj-a** [comment]
It's possible to change the Identify Magic action to have some special handling for scrolls and wands. It's a little annoying, but it's doable.
